project initial

AlexFinokhin opened the pull request.

AlexFinokhin committed:
project initial

nehydna commented:
# Резюме по работе:
## Верстка и JSX

### Критерии, влияющие на работоспособность
-   <!--12--> Все блоки и элементы из макета сверстаны. Размеры и расположение не отличаются от макета на более, чем на 30px на десктопной версии и на 10px на остальных размерах экрана, для которых подготовлен макет. Отличия в размерах не должны визуально противоречить макету и выбиваться из заложенной в макет сетки, в этом случае требуется более строгое соответствие размерам макета. 

На главной надо сделать кнопки "Войти" и "Регистрация". Мне надо посмотреть как они выглядят и как расположены.
Параметры бургера не по макету http://joxi.ru/52apXMvc0RKRbm

- ✅ <!--10.55--> Проект адаптирован под различные разрешения экрана. Горизонтальный скролл не возникает на разрешениях от 320 пикселей и больше. Нельзя скрывать полосу прокрутки свойством overflow: hidden
- ✅  <!--9.55--> Корректно работает навигация между страницами и ссылки на внешние ресурсы: ни одна ссылка не ведёт в пустоту или на якорь, внешние ссылки открываются в новой вкладке. На все страницы проекта можно попасть, а скрытые блоки можно отобразить.
- ✅ <!--5.55--> Именование классов сделаны по БЭМ
-   <!--8.55--> Отзывчивая вёрстка, которая корректно тянется на всех промежуточных разрешениях

У всех элементов должен быть отступ снизу на десктопе и на всем адаптиве. Например, http://joxi.ru/D2PngZ3tWe3E8r
Это требование критерия **на всех промежуточных разрешениях**. 
Отступ снизу не должен зависеть от высоты экрана, пользователю должно быть удобно управлять страницей на любой высоте, на любом разрешении.

- ✅  <!--3.38--> В коде используется семантическая разметка: применяются семантические теги, выбор элементов при вёрстке корректен (параграф должен быть параграфом, список — списком); структура DOM-дерева состоит не только из контейнеров div.
- ✅ <!--3.38--> Для позиционирования элементов выбран верный подход, описанный корректным синтаксисом.
- ✅ <!--3.38--> Каркас макета реализован на Flex layout или Grid layout;
- ✅ <!--3.38--> Инфраструктурные файлы проекта созданы через CRA.
- ✅ <!--3.38--> Работа модальных окон настроена.
- ✅ Разметка портирована в JSX:
	- ✅ <!--3.38--> разметка заключена в ( ), лишние пустые <div> для этого не используются;
	- ✅ <!--3.38--> разметка вынесена в соответствующие ей компоненты

-  В проекте есть:
	- ✅ <!--3.38--> директория images/ с картинками,
            - ✅ <!--3.38--> директория components/ с JS и CSS-файлами компонентов,
	- ✅ <!--3.38--> директория fonts/ со шрифтами.

### Хорошие практики
-  ✅ <!--2.14--> Кнопки, инпуты и ссылки реализованы во всех состояниях, указанных в макете. 
- ✅ <!--2.14--> В макете есть одинаковые элементы. Для их оформления должен быть переиспользован один компонент.
- ✅  <!--2.14--> Шрифты подключены через @font-face.
- ✅  <!--2.14--> Иконки экспортированы в формате SVG.
- ✅ <!--2.14--> Элементы формы должны выделяться, когда на них установлен фокус. Состояние фокуса, а также outline стилизованы в соответствии с макетом или на своё усмотрение.
- ✅ <!--2.14--> У формы должны быть заданы плейсхолдеры, валидированы обязательные поля
- ✅ <!--2.14--> reset.css запрещен.

### Рекомендации
- ✅ <!--1.68--> Для каждого шрифта указаны альтернативные варианты из системных шрифтов.
- ✅  <!--1.67--> Для изображений задан атрибут alt с подходящий значением.
- ✅ <!--1.67--> Растровые и векторные изображения оптимизированы;

## Количество баллов:  79
## Доп. Комментарии
- Если есть что-то, что не вошло в чеклист
Неверно указан атрибут lang

AlexFinokhin committed:
Fix: изменен lang с en на ru

AlexFinokhin committed:
UPD:  isLogged: true изменен на  isLogged: false

Необходимо заменить на true, чтобы проверить работоспособность залогиненного аккаунта(navigation и burger menu)

AlexFinokhin committed:
Fix: изменен размер иконки burger menu

AlexFinokhin committed:
Fix: добавлены отступы к низу на разных разрешениях

nehydna commented:
# Резюме по работе:
## Верстка и JSX

### Критерии, влияющие на работоспособность
- ✅ <!--12--> Все блоки и элементы из макета сверстаны. Размеры и расположение не отличаются от макета на более, чем на 30px на десктопной версии и на 10px на остальных размерах экрана, для которых подготовлен макет. Отличия в размерах не должны визуально противоречить макету и выбиваться из заложенной в макет сетки, в этом случае требуется более строгое соответствие размерам макета. 
- ✅ <!--10.55--> Проект адаптирован под различные разрешения экрана. Горизонтальный скролл не возникает на разрешениях от 320 пикселей и больше. Нельзя скрывать полосу прокрутки свойством overflow: hidden
- ✅ <!--9.55--> Корректно работает навигация между страницами и ссылки на внешние ресурсы: ни одна ссылка не ведёт в пустоту или на якорь, внешние ссылки открываются в новой вкладке. На все страницы проекта можно попасть, а скрытые блоки можно отобразить.
- ✅ <!--5.55--> Именование классов сделаны по БЭМ
- ✅ <!--8.55--> Отзывчивая вёрстка, которая корректно тянется на всех промежуточных разрешениях
-  ✅ <!--3.38--> В коде используется семантическая разметка: применяются семантические теги, выбор элементов при вёрстке корректен (параграф должен быть параграфом, список — списком); структура DOM-дерева состоит не только из контейнеров div.
- ✅ <!--3.38--> Для позиционирования элементов выбран верный подход, описанный корректным синтаксисом.
- ✅ <!--3.38--> Каркас макета реализован на Flex layout или Grid layout;
- ✅ <!--3.38--> Инфраструктурные файлы проекта созданы через CRA.
- ✅ <!--3.38--> Работа модальных окон настроена.
- ✅ Разметка портирована в JSX:
	- ✅ <!--3.38--> разметка заключена в ( ), лишние пустые <div> для этого не используются;
	- ✅ <!--3.38--> разметка вынесена в соответствующие ей компоненты

-  В проекте есть:
	- ✅ <!--3.38--> директория images/ с картинками,
            - ✅ <!--3.38--> директория components/ с JS и CSS-файлами компонентов,
	- ✅ <!--3.38--> директория fonts/ со шрифтами.

### Хорошие практики
-  ✅ <!--2.14--> Кнопки, инпуты и ссылки реализованы во всех состояниях, указанных в макете. 
- ✅ <!--2.14--> В макете есть одинаковые элементы. Для их оформления должен быть переиспользован один компонент.
- ✅  <!--2.14--> Шрифты подключены через @font-face.
- ✅  <!--2.14--> Иконки экспортированы в формате SVG.
- ✅ <!--2.14--> Элементы формы должны выделяться, когда на них установлен фокус. Состояние фокуса, а также outline стилизованы в соответствии с макетом или на своё усмотрение.
- ✅ <!--2.14--> У формы должны быть заданы плейсхолдеры, валидированы обязательные поля
- ✅ <!--2.14--> reset.css запрещен.

### Рекомендации
- ✅ <!--1.68--> Для каждого шрифта указаны альтернативные варианты из системных шрифтов.
- ✅  <!--1.67--> Для изображений задан атрибут alt с подходящий значением.
- ✅ <!--1.67--> Растровые и векторные изображения оптимизированы;

## Количество баллов:  100
## Доп. Комментарии
- Если есть что-то, что не вошло в чеклист
